reference/docs-conceptual/components/vscode/using-vscode.md

```diff
@@ -83,27 +83,83 @@ You are prompted with "Do you want to run software from this untrusted publisher
 Type `R` to run the file. Then, open Visual Studio Code and check that the PowerShell extension is
 functioning properly. If you still have issues getting started, let us know on [GitHub](https://github.com/PowerShell/vscode-powershell/issues).
 
-#### Using a specific installed version of PowerShell
+#### Choosing a version of PowerShell to use with the extension
 
-If you wish to use a specific installation of PowerShell with Visual Studio Code, you need to add a new variable to your user settings file.
+With PowerShell Core installing side-by-side with Windows PowerShell, is it now possible to a
```

Comment:
Sorry, late to this but aren't we missing the word `use` at the end of this line?

Reply:
@rkeithhill Thanks for catching that. I have fixed it now.